R/list-combine.R

```diff
@@ -58,19 +66,26 @@ list_cbind <- function(
     x,
     ...,
     name_repair = c("unique", "universal", "check_unique"),
-    size = NULL
+    size = NULL,
+    keep_empty = FALSE
```

Comment:
@DavisVaughan my intuition is that `keep_empty` doesn't make sense for `list_cbind()`, but I don't have anything concrete to back that up. Do you have any thoughts?